webxrlayers-1.bs

```diff
@@ -348,6 +349,11 @@ enable optical chromatic aberration correction for the layer. If the user agent
 should return <code>null</code> on getting, and setting should be a <code>no-op</code>.
 If the {{XRCompositionLayer/chromaticAberrationCorrection}} was changed, it will take effect at the next {{XRFrame}}.
 
+The <dfn attribute for="XRCompositionLayer">forceMonoPresentation</dfn> attribute forces the right eye to use the same layer configuration
+as the left eye. This MAY be used to remove the {{XRLayerLayout/stereo}} effect of a {{XRCompositionLayer}}. Setting this to <code>true</code> on a {{XRLayerLayout/mono}} layer has no effect.
```

Comment:
LGTM but with a question that can be addressed in a followup if needed, since this is already merged.

Forcing mono presentation seems important enough for some uses cases that it feels like they might want to switch to an alternate rendering path (like manually blitting the video to a WebGL layer) if the mono effect doesn't work here. The use of "MAY" here feels like it opens up the implementation to some interpretation. I'd prefer a stronger guarantee if we can make it, such that if this attribute is present in an implementation it will always take effect.

Reply:
Maybe it can be rephrased to "This will tell the xr compositor to remove the stereo effect of a composition layer"?

Reply:
That  works!